### PR TITLE
Optionally remove headlines from feed items on the JS Widget

### PR DESCRIPTION
**Pull request by urfolomeus**
Some partners use the JS Widget within a CMS that already has a title field
that is used to put in the headline of the feed item. As such we can optionally
serve that content without the headlines.

### CODE REVIEW

File: app/helpers/feed_helper.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module FeedHelper
-  def script_tag_for(feed)
-    source = widget_feed_url(feed, protocol: '//', format: :js)
+  def script_tag_for(feed, exclude_headlines)
```
Comment (asmega) [inline app/helpers/feed_helper.rb]: I'd use an options hash here so the api is more flexible to change

Comment (urfolomeus) [inline app/helpers/feed_helper.rb]: A good point. I'll make that change. Thanks.

Comment (asmega) [inline app/helpers/feed_helper.rb]: or named argument

**Comment by munckymagik** [comment]
:+1:

**Comment by urfolomeus** [comment]
@asmega implemented as suggested.

**Comment by asmega** [comment]
:+1:

**Pull request merged**